src/config.rs

```diff
@@ -730,6 +738,15 @@ impl Config {
             .unwrap_or(false)
     }
 
+    #[cfg(target_os = "linux")]
+    pub fn firmware_upgrade(&self) -> bool {
+        self.config_file
+            .firmware
+            .as_ref()
+            .and_then(|firmware| firmware.upgrade)
+            .unwrap_or(true)
```

Comment:
I think that the default should be false since it's a dangerous operation

Reply:
I agree. `true` matches the current behavior, but `false` makes more sense :smile: